pkgs/development/python-modules/tubes/default.nix

```
@@ -1,19 +1,30 @@
-{ lib, buildPythonPackage, fetchPypi, python
-, characteristic, six, twisted
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  python,
+  characteristic,
+  six,
+  twisted,
 }:
 
 buildPythonPackage rec {
   pname = "tubes";
-  version = "0.2.1";
+  version = "b74680b8e7bcfe64362865356bb9461b77bbd5c0";
```

```suggestion
  version = "0.2.1-unstable-2023-11-06";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming